src/uct/gaudi/base/gaudi_topo.c

```diff
@@ -34,66 +34,74 @@
 #define UCT_GAUDI_TOPO_MELLANOX_VENDOR_ID  0x15b3
 #define UCT_GAUDI_TOPO_BROADCOM_VENDOR_ID  0x14e4
 #define GAUDI_DEVICE_NAME_LEN              10
 
-static pthread_mutex_t gaudi_init_mutex = PTHREAD_MUTEX_INITIALIZER;
+static pthread_mutex_t uct_gaudi_init_mutex        = PTHREAD_MUTEX_INITIALIZER;
 /* File-scope one-time control and status */
-static pthread_once_t gaudi_spinlock_once_flag = PTHREAD_ONCE_INIT;
-static ucs_status_t gaudi_spinlock_init_status = UCS_OK;
-
-static const ucs_sys_dev_distance_t gaudi_fallback_node_distance =
-        {.latency = 100e-9, .bandwidth = 17e9}; /* 100ns, 17 GB/s */
-static const ucs_sys_dev_distance_t gaudi_fallback_sys_distance =
-        {.latency = 300e-9, .bandwidth = 220e6}; /* 300ns, 220 MB/s */
+static pthread_once_t uct_gaudi_spinlock_once_flag = PTHREAD_ONCE_INIT;
+static ucs_status_t uct_gaudi_spinlock_init_status = UCS_OK;
+
+static const ucs_sys_dev_distance_t uct_gaudi_fallback_node_distance = {
+    .latency   = 100e-9,
+    .bandwidth = 17e9
+}; /* 100ns, 17 GB/s */
+static const ucs_sys_dev_distance_t uct_gaudi_fallback_sys_distance  = {
+    .latency   = 300e-9,
+    .bandwidth = 220e6
+}; /* 300ns, 220 MB/s */
 
 /* Structure to hold Gaudi and HNIC mappings */
 typedef struct {
-    ucs_sys_device_t gaudi_device;
-    ucs_sys_device_t hnic_device;
-    uint16_t hnic_vendor_id;
+    ucs_sys_device_t       gaudi_device;
+    ucs_sys_device_t       hnic_device;
+    uint16_t               hnic_vendor_id;
     ucs_sys_dev_distance_t distance;
-    ucs_numa_node_t common_numa_node;
-    char gaudi_dev_name[GAUDI_DEVICE_NAME_LEN];
+    ucs_numa_node_t        common_numa_node;
+    char                   gaudi_dev_name[GAUDI_DEVICE_NAME_LEN];
 } uct_gaudi_connection_t;
 
 /* Static context for Gaudi topology */
+// clang-format off
 typedef struct {
-    ucs_spinlock_t lock;
-    unsigned initialized;
-    unsigned provider_added;
-    ucs_sys_device_t *gaudi_devices;
-    char (*gaudi_devices_names)[GAUDI_DEVICE_NAME_LEN];
-    unsigned num_gaudi_devices;
-    ucs_sys_device_t *hnic_devices;
-    uint16_t *hnic_vendor_ids;
-    unsigned num_hnic_devices;
+    ucs_spinlock_t         lock;
+    unsigned               initialized;
+    unsigned               provider_added;
+    ucs_sys_device_t       *gaudi_devices;
+    char                   (*gaudi_devices_names)[GAUDI_DEVICE_NAME_LEN];
+    unsigned               num_gaudi_devices;
+    ucs_sys_device_t       *hnic_devices;
+    uint16_t               *hnic_vendor_ids;
+    unsigned               num_hnic_devices;
     uct_gaudi_connection_t *connections;
-    unsigned num_connections;
-    ucs_sys_device_t *assigned_hnic_for_gaudi;
-    int *assigned_port_for_gaudi;
-    unsigned have_assignment;
+    unsigned               num_connections;
+    ucs_sys_device_t       *assigned_hnic_for_gaudi;
+    int                    *assigned_port_for_gaudi;
+    unsigned               have_assignment;
 } uct_gaudi_topo_ctx_t;
+// clang-format on
```

Comment:
pls don't use c++ comments (remove foramt-off , we will ignore the suggestion if it is not applicable)